client/src/components/Chat.jsx

```diff
@@ -535,6 +535,8 @@ const Chat = () => {
 																cancelEdit={cancelEdit}
 																setEditing={setEditing}
 																setReplyId={startReply}
+																cryptoKey={cryptoKey}
+																importedPrivateKey={importedPrivateKey}
```

Comment:
No need to pass it down, it's a Hook that can be imported in the dropdownoption component.

Here: https://github.com/Dun-sin/Whisper/blob/c6f0687bbb759262e99d23e8ffb40eb0ca9c6afd/client/src/hooks/useCryptoKeys.js#L8

So just import it and then use the needed functions

Like this: https://github.com/Dun-sin/Whisper/blob/c6f0687bbb759262e99d23e8ffb40eb0ca9c6afd/client/src/components/Chat.jsx#L70